### PR TITLE
feat(react-storybook-addon): improve addon to more readable names

### PR DESCRIPTION
**Pull request by marcosmoura**
Apply some simple fixes:
- Improve addon title to be more readable
- Remove Griffel as a dependency

### CODE REVIEW

File: packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { IconButton } from '@storybook/components';
+
+import { DIR_ID } from '../constants';
+import { useGlobals } from '../hooks';
+
+const monospaceStyle = {
```
Comment (Hotell) [inline packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx:7]: for this use case I guess this is ok but going forward we should use storybook apis (emotion)

```
import { styled } from '@storybook/theming';
...
```

see https://storybook.js.org/docs/addons/addon-knowledge-base#style-your-addon

**Review by Hotell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-storybook-addon/src/preset/manager.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { addons, types } from '@storybook/manager-api';
 import { ADDON_ID, DIR_ID, STRICT_MODE_ID, THEME_ID } from '../constants';
 import { ThemePicker } from '../components/ThemePicker';
 import { ReactStrictMode } from '../components/ReactStrictMode';
-import { DirSwitch } from '../components/DirSwitch';
```
Comment (Hotell) [inline packages/react-components/react-storybook-addon/src/preset/manager.ts:6]: for whatever reason this is not shown as rename :D , so if you did not rename make sure to remove the DirSwitch file . ty

File: packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx (hunk 1 above)
Comment (marcosmoura) [reply packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx:7]: I thought this implementation was Storybook 8 exclusive. I updated the code to use Emotion.

File: packages/react-components/react-storybook-addon/src/preset/manager.ts (hunk 2 above)
Comment (marcosmoura) [reply packages/react-components/react-storybook-addon/src/preset/manager.ts:6]: 👀

File: packages/react-components/react-storybook-addon/package.json [hunk 3]
```diff
@@ -33,7 +33,6 @@
     "@fluentui/react-aria": "^9.13.5",
```
Comment (fabricteam) [inline packages/react-components/react-storybook-addon/package.json:33]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32815&commitId=57c41336c8c6c5f32bc065cfaff197c08900fdd5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|988| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx (hunk 1 above)
Comment (Hotell) [reply packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx]: yeah navigating in storybook related things is challenging . ty!

**Pull request merged**